### DISCUSSION TITLE
3.5.1 release

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#3792] [Announcements]
- fixed a crash in uninstaller
- disable lazy loading on session restore

<hr /><em>This discussion was created from the release <a href='https://github.com/sumatrapdfreader/sumatrapdf/releases/tag/3.5.1rel'>3.5.1 release</a>.</em>

**Comment by AnonymousWP** [comment]
First of all: thank for the great work! Seems like there's no asset/installer available yet. Also, another question: do you plan to release more frequent updates (as in smaller updates but more frequent instead of 1 big update every year for example)? e.g. it has been over a year when you compare 3.5.0 to 3.4.6.

**Reply by GitHubRulesOK** [reply]
@AnonymousWP the runtime is the installer/uninstaller it is a "ONE EXE" self installable application not unlike others that morph from win console CLI to Win GUI at different times, so no need for any separate variants other than a PORTABLE single EXE so thats 6 variations without more.
Smaller updates have been running daily in many cases so thats 6 signed copies needed per day ! official signing is not free. and AFAIK there is an overhead for running compilations too.

**Reply by AnonymousWP** [reply]
Right, so where do I download 3.5.1 then?

**Reply by kjk** [reply]
@AnonymousWP 3.5.1 should now be available

As to more frequent releases: maybe. It all depends on how much time I have to spend on SumatraPDF.

There's always pre-release build (https://www.sumatrapdfreader.org/prerelease) with latest changes. Upgrading is really easy. They occasionally have bugs non present in latest release but also have bug fixes not present in latest release.

**Reply by kjk** [reply]
@AnonymousWP https://www.sumatrapdfreader.org/download-free-pdf-viewer just make sure to hard refresh.

Or directly: https://www.sumatrapdfreader.org/dl/rel/3.5.1/SumatraPDF-3.5.1-64-install.exe

I'm using cloudflare to speed up access and it might return cached pages. In Chrome try `Ctrl + F5` or press `Ctrl` when clicking reload button (https://fabricdigital.co.nz/blog/how-to-hard-refresh-your-browser-and-clear-cache)

**Reply by AnonymousWP** [reply]
Ah, thanks, yeah I see 3.5.1 being available on the website now. I see, I will consider pre-releases if I 'need' to (dunno, features I'd really like for example).

**Comment by VsevolodGolovanov** [comment]
Thank you for your work, I'm a long time user :)
Is there a problem with lazy loading? Will it come back? It's probably my 2nd most wanted feature after subpixel rendering (I haven't noticed that 3.5.0 had it, I'm still on 3.4.6.)

**Reply by kjk** [reply]
this could be a cause of (perceived?) slowness in https://github.com/sumatrapdfreader/sumatrapdf/issues/3791 
It's one of those things where it's hard to please everyone because there are tradeoffs.
Maybe will do a setting for that.

**Comment by tossnet** [comment]
Hi ! Bravo! 
since 3.5, I don't know if you've seen but there's a bigger rendering (tab)...
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/3845786/15dc4909-0910-414d-9d8c-7db44e839e5e)

**Reply by GitHubRulesOK** [reply]
seems to be a windows 11 problem here is windows 10
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/901ea131-684e-4ecb-b339-6421bb34dfec)

**Reply by tossnet** [reply]
No isn't Windows 11 problem / Windows 10 . I've been working on Windows 11 since the beginning and I update Sumatra as soon as possible and the problem appeared in v3.5. (cc @kjk )

**Comment by kjk** [comment]
We follow DPI scaling so bigger tabs are expected if windows Display Settings scaling is set to >100%.

So the question is:
- is it different than previous version of Sumatra?
- if yes, how does Sumatra compares to other apps like Chrome. Do they have smaller / bigger tabs etc.?
- any customization of Sumatra? There are settings to change sizes of things

**Reply by tossnet** [reply]
🖖

1. yes
2. only sumatra
3. no. I don't touch any parameeters of Sumatra

**Comment by Netkuszkusz** [comment]
Still crashing with new file name:

![2352](https://github.com/sumatrapdfreader/sumatrapdf/assets/15225482/63f94bb3-c7d6-4715-a594-efdb5aafd6d0)

This is just pure stuipidity. Without any recognisable arguments the EXE should work as basic installer instead of crashing. Can you please set this "feature" up as ANY windows installer did so far on Earth through history? Thanks.
